src/software/utils/main_imageProcessing.cpp

```diff
@@ -183,9 +183,9 @@ int aliceVision_main(int argc, char * argv[])
         return EXIT_FAILURE;
     }
 
-      // Analyze output path
-    const boost::filesystem::path path(sfmOutputDataFilepath);
-    const std::string outputPath = path.parent_path().string();
+    // check if is sfm data file
+    if(fs::path(sfmInputDataFilename).extension().string() == ".abc")
```

Comment:
add .sfm extension

Reply:
Fixed in https://github.com/alicevision/AliceVision/pull/806/commits/4670526623ce19e5c00f8cd128f8eaebc9acd934 .